discovery-provider/src/queries/notifications.py

```diff
@@ -726,6 +731,66 @@ def notifications():
             track_id = entry.track_id
             owner_info[const.tracks][track_id] = owner
 
+        # Get playlist updates
+        today = date.today()
+        thirty_days_ago = today - timedelta(days=30)
+        thirty_days_ago_time = datetime(
+            thirty_days_ago.year,
+            thirty_days_ago.month,
+            thirty_days_ago.day,
+            0,
+            0,
+            0
+        )
+        playlist_update_query = session.query(Playlist)
+        playlist_update_query = playlist_update_query.filter(
+            Playlist.is_current == True,
+            Playlist.is_delete == False,
+            Playlist.blocknumber > min_block_number,
+            Playlist.blocknumber <= max_block_number)
+
+        playlist_update_results = playlist_update_query.all()
+
+        # Represents all playlist update notifications
+        playlist_update_notifications = []
+        playlist_update_notifs_by_playlist_id = {}
+        for entry in playlist_update_results:
+            if entry.last_added_to < thirty_days_ago_time:
```

Comment:
could we include this filter in the query itself rather than just a pass over here?